android/app/build.gradle

```diff
@@ -1,8 +1,21 @@
+def taskRequests = getGradle().getStartParameter().getTaskRequests().toString()
+def isPlay = !(taskRequests.contains("Foss") || taskRequests.contains("foss"))
+
 apply plugin: "com.android.application"
 apply plugin: 'kotlin-android'
-apply plugin: "io.fabric"
-apply plugin: "com.google.firebase.firebase-perf"
+
 apply plugin: 'com.bugsnag.android.gradle'
+apply plugin: 'com.google.gms.google-services'
```

Comment:
Is that supposed to be inside `isPlay`?

Reply:
I earlier thought that disabling google Services would be a better option instead of disabling the plugin.  Will update it 👍